app/views/csvfiles/new.html.erb

```diff
@@ -46,6 +46,9 @@
                             filestring: csvStringForDownload},
                     success: function(data){
                         console.log(data,"saved");
+                    },
+                    error: function(xhr){
+                        alert("Couldn't save this file");
```

Comment:
An error function in an ajax takes 3 arguments, `XMLHttpRequest, textStatus, errorThrown`. Instead of a javascript alert, it's better to stay consistent and use different types of bootstrap alerts already used in the project. A better message will be `There was some error saving the file. Please try again." Also I might suggest logging the error so that if someone checks the console then they can report it using the issue tracker.

Reply:
Hi @IshaGupta18 Here Anupam-dagar has given correct explanation. We have three arguments in error. Maybe we can have a some better message.